mpqp/core/instruction/gates/native_gates.py

```diff
@@ -296,6 +296,8 @@ def qiskit_gate(cls):
 
         return IGate
 
+    qasm2_gate = "id"
```

Comment:
if we move this to the instr constructor, we could have this in the `uper().__init__`